docs/TOKENS.md

```diff
@@ -3,7 +3,7 @@ title: Build a Token Generator with Astro
 description: Learn how you can generate tokens using SSR frameworks like Astro.
 ---
 
-Astro is created to be the perfect use case for a content-heavy website. A content heavy site is mostly front end code. The opposite of that is a backend server, and in this guide, we will build a backend token generator for Agora video calls. 
+Astro is created to be the perfect use case for a content-heavy website, which is mostly front-end code. The opposite of that is a backend server, and in this guide, we will build a backend token generator for Agora video calls. 
```

Comment:
In the first sentence the latest update makes the idea less clear. The explicit connection is now implicit but not clear as to where the connection leads. When you say "which is mostly front-end code", its unclear if you are referring to Astro or the content-heavy website. 

> which mostly consists of front-end code.

In the second sentence the "that" is no longer immediately clear. Again the connection is broken as to what is the opposite of the "backend server". Is it Astro or a content heavy site or is it the front-end? Consider adding some clarity to the statement.

Reply:
I changed it back to how it was